specification/app/resource-manager/Microsoft.App/preview/2023-08-01-preview/Patches.json

```diff
@@ -0,0 +1,542 @@
+              "$ref": "./CommonDefinitions.json#/definitions/DefaultErrorResponse"
+            }
+          }
+        },
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "location"
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-examples": {
+          "Patches_Skip_Configure_0": {
+            "$ref": "./examples/Patches_Skip_Configure.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.App/builders/{builderName}/patches/{patchName}/apply": {
+      "post": {
+        "tags": [
+          "Patches"
+        ],
+        "operationId": "Patches_Apply",
+        "description": "Apply a PatchResource with patch name.",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "./Builders.json#/parameters/BuilderNameParameter"
+          },
+          {
+            "$ref": "#/parameters/PatchNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Patching operation completed successfully.",
+            "schema": {
+              "$ref": "#/definitions/PatchResource"
+            }
+          },
+          "202": {
+            "description": "Patching operation has been started.",
+            "schema": {
+              "$ref": "#/definitions/PatchResource"
+            },
+            "headers": {
+              "Azure-AsyncOperation": {
+                "type": "string"
+              }
+            }
+          },
+          "default": {
+            "description": "Common error response.",
+            "schema": {
+              "$ref": "./CommonDefinitions.json#/definitions/DefaultErrorResponse"
+            }
+          }
+        },
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "azure-async-operation"
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-examples": {
+          "Patches_Apply_0": {
+            "$ref": "./examples/Patches_Apply.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "PatchResource": {
+      "type": "object",
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/PatchProperties",
+          "description": "The resource-specific properties for this resource.",
+          "x-ms-client-flatten": true,
+          "x-ms-mutability": [
+            "read",
+            "create"
+          ]
+        }
+      },
+      "description": "The resource representation of a patch.",
```

Comment:
> "description": "The resource representation of a patch.", [](http://example.com/codeflow?start=5&length=58)

This kind of roundabout description is discouraged. Instead of saying its a 'representation' of a patch, why not say its 'a patch'?